src/release/breaking-changes/navigator-and-page-api.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Navigator's page APIs breaking change
+description: >
+  Changes to Navigator's page APIs.
+---
+
+## Summary
+
+The [`Navigator`]'s page APIs are refactored so that they can integrate
+into Flutter's other pop mechanisms.
+
+## Context
+
+The `onPopPage` was added for cleaning up pages after a page is about to be popped.
+The developer can veto pop by returning false in the callback. This does not work well
```

Comment:
**suggestion:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). 
Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To veto pop, the developer can return false in the callback. This does not work well
```